x/config/config.go

```diff
@@ -77,6 +77,10 @@ func NewDefaultConfigToDialer() *ConfigToDialer {
 		}
 		return tlsfrag.NewFixedLenStreamDialer(sd, fixedLen)
 	})
+
+	p.RegisterStreamDialerType("ws", wrapStreamDialerWithWebsocket)
```

Comment:
*nit* [IETF uses `WebSocket`](https://datatracker.ietf.org/doc/html/rfc6455) (uppercase 'S') instead of `Websocket` (lowercase 's'), shall we align the code name with it?

```suggestion
	p.RegisterStreamDialerType("ws", wrapStreamDialerWithWebSocket)
```